extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -150,11 +239,28 @@ export class DatabaseSettingsPage extends BasePage {
 		});
 	}
 
+	//Collation name has no rules, aside from it not being all spaces (No REST APIs exist for finding the list).
```

Comment:
Don't put single line comments like this above function names - put it inside if the comment isn't relevant as a doc comment